portal/src/util/deno.ts

```diff
@@ -1,3 +1,3 @@
-const authgear_deno_hook_version = "v1.4.0";
+const authgear_deno_hook_version = "v1.5.0";
```

Comment:
You also need to update the template in SMSProviderConfigurationScreen to use the new response type.

Reply:
oh right